packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -530,19 +529,22 @@ export default class New extends Component {
   _probe = async (host, type) => {
     const probeMethodFactories = {
       hba: async hostId => ({
-        hbaDevices: await probeSrHba(hostId)::ignoreErrors(),
+        hbaDevices: await probeSrHba(hostId),
       }),
       zfs: async hostId => ({
-        zfsPools: await probeZfs(hostId)::ignoreErrors(),
+        zfsPools: await probeZfs(hostId),
       }),
     }
     if (probeMethodFactories[type] !== undefined && host != null) {
       this.setState(({ loading }) => ({ loading: loading + 1 }))
-      const probeResult = await probeMethodFactories[type](host.id)
-      this.setState(({ loading }) => ({
-        loading: loading - 1,
-        ...probeResult,
-      }))
+      try {
+        const probeResult = await probeMethodFactories[type](host.id)
+        this.setState({ ...probeResult })
```

Comment:
You don't need to spread the variable
```suggestion
        this.setState(probeResult)
```